plugins/stern.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: stern
+spec:
+  version: v1.20.1
+  platforms:
+  - bin: stern.exe
+    uri: https://github.com/stern/stern/releases/download/v1.20.1/stern_1.20.1_windows_amd64.tar.gz
+    sha256: 3ed6ccd050a72b68faf2f8f931969aabbd4b69fa69e1abe7ca4aaa2cf4889c22
+    selector:
+      matchLabels:
+        os: windows
+        arch: amd64
+    files:
+    - from: stern_1.20.1_windows_amd64/*
+      to: .
+  - bin: stern
+    uri: https://github.com/stern/stern/releases/download/v1.20.1/stern_1.20.1_linux_arm64.tar.gz
+    sha256: 194bbdf77a9eaee71fa830c59db5bd9af8148d710fbe64dc656ab505ad2f66c3
+    selector:
+      matchLabels:
+        os: linux
+        arch: arm64
+    files:
+    - from: stern_1.20.1_linux_arm64/*
+      to: .
```

Comment:
it's highly advised you dont put version names in the file paths. otherwise, the krew-index auto-approval procedure that we use for version bumps aren't gonna work. please consider using globs (*) here for the versions.

Reply:
I will consider to stop wrapping a binary in a directory in the next release of stern.